tardis_kromer_plot.py

```diff
@@ -24,11 +23,11 @@
 
 logger = logging.getLogger(__name__)
 
-with open('elements.csv') as f:
-    reader = csv.reader(f, skipinitialspace=True)
```

Comment:
It's not necessary because you use a with statement, but I like to be in the habit of manually closing opened files when I'm done with them.

Reply:
yeah so this was the old way, which I guess wasn't the best way to do it - the new method is more streamlined, and uses everyone's favourite tool - pandas!